go-controller/pkg/ovn/gateway_init.go

```diff
@@ -584,6 +569,101 @@ func addPolicyBasedRoutes(nodeName, mgmtPortIP string, hostIfAddr *net.IPNet) er
 	return nil
 }
 
+// This function sync logical router policies which concern the management port.
+// This function compares the following ovn-nbctl output:
+
+// 		72db5e49-0949-4d00-93e3-fe94442dd861,ip4.src == 10.244.0.2 && ip4.dst == 172.18.0.2 /* ovn-worker2 */,169.254.0.1
+// 		c20ac671-704a-428a-a32b-44da2eec8456,"inport == ""rtos-ovn-worker2"" && ip4.dst == 172.18.0.2 /* ovn-worker2 */",10.244.0.2
```

Comment:
@alexanderConstantinescu OVN has had external-ids for the Logical_Router_Policy table since 20.06, which we're long past requiring. Does it make sense to use those instead of comments now?

Reply:
So, I as I mentioned here, https://github.com/ovn-org/ovn-kubernetes/blob/ededa4995ca2a766bcbad28a915b549293006ea3/go-controller/pkg/ovn/gateway_init.go#L589-L592

We could do that, but since we don't version OVN-Kubernetes...when would we know that everyone using OVN-Kubernetes has upgraded to a version running that code, and thus being able to remove everything based on the encoded characters in the match condition and fully use the `external_id`?